src/card/skin.css

```diff
@@ -95,3 +95,25 @@
     color: var(--spectrum-card-quiet-subtitle-text-color);
   }
 }
+
+:focus-ring {
```

Comment:
Does this need to be under `:focus-ring` at all? Shouldn't we just leave it as `.spectrum-Card.is-focused` programmatically when the grid cell is focused and be done?

Reply:
It should be `:focus-ring` otherwise, the focused item in the grid will render with blue focus ring even on touch devices or when a user deselects a Card using the mouse.